p/themes/base-theme/template.css

```diff
@@ -1135,7 +1135,7 @@ input:checked + .slide-container .properties {
 	margin-left: .8em;
 }
 
-pre.enclosure-description {
+.enclosure-description {
 	white-space: pre-line;
```

Comment:
Does this style still make sense if you're just using a regular `<p>`?

Reply:
I think this can be applied to any block, including div, p, etc.

Reply:
Sure, I just meant to double check if it was providing added value. :+1: